test/powershell/Modules/Microsoft.PowerShell.Utility/Tee-Object.Tests.ps1

```diff
@@ -18,7 +18,24 @@ Describe "Tee-Object" -Tags "CI" {
 	        Write-Output teeobjecttest3  | Tee-Object $teefile
 	        Get-Content $teefile | Should -BeExactly "teeobjecttest3"
 	        Remove-Item $teefile -ErrorAction SilentlyContinue
-	    }
+        }
+
+        It "Should tee output to file using selected encoding" -TestCases @(
+            @{ Encoding = "ascii" },
+            @{ Encoding = "bigendianunicode" },
+            @{ Encoding = "default" },
+            @{ Encoding = "latin1" },
+            @{ Encoding = "unicode" },
+            @{ Encoding = "utf32" },
+            @{ Encoding = "utf7" },
+            @{ Encoding = "utf8" }
+        ) {
+            param($Encoding)
+            $teefile = $testfile
+            Write-Output -InputObject teeobjecttest3  | Tee-Object -FilePath $teefile -Encoding $Encoding
```

Comment:
Really the test doesn't check encoding because write ASCII only.
Please see `Verifies -Tail with content that uses an explicit/implicit encoding` from Get-Content.Tests.ps1 . We could use the same string with native chars instead of `teeobjecttest3`.